index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kenyan Recipes</title>
+
+  <link rel="stylesheet" href="css/styles.css"/>
+
+</head>
+<body>
+
+    <h1 class="heading-text">Kenya's Best Meals</h1>
+    <p>The three best Kenyan recipes are ugali, Mukimo and Matoke. </p>
+    <p>Click the links below for detailed recipes on each of the meals</p>
+
+    <ul>
+        <li><a href="kenyan-recipes/mokimo.html">Mokimo Recipe</a></li>
+        <li><a href="kenyan-recipes/ugali.html">Ugali Recipe</a></li>
+        <li><a href="kenyan-recipes/matoke.html">Matoke Recipe</a></li>
+
+    </ul>
+    
+    <img src="images/people smiling.jpg" alt="people eating ugali" width="" height="">
```

Comment:
Use semantic tags in your HTML code. You can read more on semantic tags https://www.w3schools.com/html/html5_semantic_elements.asp

Reply:
- [ ] Always make sure to include an explanation, a "why", for each requested change, and use polite words like `Kindly`, and `It would be better`. Kindly Implement it. 👍

Reply:
@roshan-bajgain , please note, I have made the correction on my previous comment

Reply:
Hi, @kiboma2021 !

I notice that you applied the requested changes and edited your original comment. Great job! :+1: